versioneer.py

```diff
@@ -0,0 +1,1822 @@
+
+# Version: 0.18
+
+"""The Versioneer - like a rocketeer, but for versions.
+
+The Versioneer
+==============
+
+* like a rocketeer, but for versions!
+* https://github.com/warner/python-versioneer
+* Brian Warner
+* License: Public Domain
```

Comment:
Is this License compatible with greenplumpython?

Reply:
Yes, it is compatible with apache license
https://creativecommons.org/publicdomain/zero/1.0/